### PR TITLE
IAAS-10398: Add support to use six library

### PR DESCRIPTION
**Pull request by shahpratikr**
Component build: http://teamcity.platform9.horse:8111/viewLog.html?buildId=1199610&tab=buildResultsDiv&buildTypeId=Pf9project_Platform9Components_Pf9ha

### CODE REVIEW

File: host/ha/pf9app/opt/pf9/etc/pf9-consul/conf.d/server.json.template [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "bootstrap_expect": 3,
+    "bootstrap_expect": "3",
```
Comment (anupbarve) [inline host/ha/pf9app/opt/pf9/etc/pf9-consul/conf.d/server.json.template:2]: Why is this needed ?

Comment (shahpratikr) [reply host/ha/pf9app/opt/pf9/etc/pf9-consul/conf.d/server.json.template:2]: Python 3 expects all strings values.

File: host/ha/pf9app/config [hunk 2]
```diff
@@ -47,23 +45,22 @@ def get_config(option, opt_str, value, parser):
...
     sys.exit(0)
 
+
 def _run_command(cmd, env=None, exit=True):
-    proc = subprocess.Popen(cmd, shell=True,
-                            stdin=subprocess.PIPE,
-                            stdout=subprocess.PIPE,
-                            stderr=subprocess.PIPE,
-                            env=env)
+    proc = subprocess.Popen(cmd, shell=True, stdin=subprocess.PIPE, env=env,
```
Comment (poojaghumre) [inline host/ha/pf9app/config:50]: change in param order not needed
